README.md

```diff
@@ -24,6 +24,81 @@ This cookbook requires Chef 11.12.0+.
+enable_non_microsoft_updates |Allows signed non-Microsoft updates.                                                                    |TrueClass, FalseClass|`true`
+allow_user_to_install_updates|Authorizes Users to approve or disapprove updates.                                                      |TrueClass, FalseClass|`false`
+auto_install_minor_updates   |Defines whether minor update should be automatically installed.                                         |TrueClass, FalseClass|`false`
+no_reboot_with_logged_users  |Disables automatic reboot with logged-on users.                                                         |TrueClass, FalseClass|`true`
+automatic_update_behavior    |Defines auto update behavior.                                                                           |Symbol`*`            |`:disabled`
+detection_frequency          |Defines times in hours between detection cycles.                                                        |FixNum               |`22`
+schedule_install_day         |Defines the day of the week to schedule update install.                                                 |Symbol`**`           |`:every_day`
+schedule_install_time        |Defines the time of day in 24-hour format to schedule update install.                                   |FixNum (1-23)        |`0`
+schedule_retry_wait          |Defines the time in minutes to wait at startup before applying update from a missed scheduled time      |FixNum (0-60)        |`0`
+reboot_warning               |Defines time in minutes of the restart warning countdown after reboot-required updates automatic install|FixNum (1-30)        |`5`
+reboot_prompt_timeout        |Defines time in minutes between prompts for a scheduled restart                                         |FixNum (1-1440)      |`10`
+
+`*` automatic_update_behavior values are:
+```ruby
+# :disabled  = Disables automatic updates
+# :detect    = Only notify users of new updates
+# :download  = Download updates, but let users install them
+# :install   = Download and install updates
+# :manual    = Lets the users configure the behavior
+```
+`**` schedule_install_day possible values are: `:every_day`, `:sunday`, `:monday`, `:tuesday`, `:wednesday`, `:thursday`, `:friday`, `:saturday`
+
+## wsus-client::update
+This recipe performs a synchronous detection and install of available Windows updates.
+
+### Attributes
+The following attributes are used to configure the `wsus-client::update` recipe, accessible via `node['wsus_client'][attribute]`.
+
+Attribute      | Description                                                          | Type   | Default
+---------------|----------------------------------------------------------------------|--------|--------
+update_timeout | The maximum amount of time the synchronous update is allowed to run. | FixNum | `7200`
+
+> **Note:** mixlib-shellout < 2.0.1 is not terminating the process after the allowed timeout. Chef 12.0.4 include the new version.
```

Comment:
**Note:** mixlib-shellout < 2.0.1 does not terminate the process after the allowed timeout. Chef > 12.0.4 includes the new version.